framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -0,0 +1,397 @@
+#
+---
+_include: tlvf/WSC/eWscLengths.h
+_namespace: WSC
+
```

Comment:
_is_tlv_class is a class attribute, while this file defined structs.
It's more convenient to work with structs in TLVF code, and because these attributes don't stand by themselves (but only as a specific part of a frame), and usually have const values, there is no real benefit in defining them as classes.

Reply:
@tomereli or @rammarzin could you confirm?